scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -257,6 +257,7 @@ cdef dict _unpack_dist(object dist, str dist_type, list meths,
                     (self.args, self.loc,
                      self.scale) = dist.dist._parse_args(*dist.args,
                                                          **dist.kwds)
+                    self.support = dist.support
```

Comment:
One last change: Added the `support` method in the `wrap_dist` class so that methods like `DiscreteAliasUrn` can work without explicitly passing a domain parameter. Also added a test for it.